src/views/SpaceSettings.vue

```diff
@@ -272,7 +272,7 @@ watchEffect(async () => {
 });
 
 onMounted(() => {
-  setPageTitle('page.title.space.settings', { space: props.space.name });
+  setPageTitle('page.title.space.settings', { space: props.space?.name });
```

Comment:
With this change the title is then " Settings" if there is no space, but ideally it should be "Settings"

Reply:
```suggestion
  setPageTitle('page.title.space.settings', { space: props.space?.name ?? '' });
```
Would this change that?

Reply:
Not sure how you can see the extra spacing